source/tests/system/niscope_restricted_driver_api_tests.cpp

```diff
@@ -84,6 +92,40 @@ TEST_F(NiScopeRestrictedDriverApiTests, NiScopeRestrictedGetStartTimestampInform
   EXPECT_EQ(0, response.device_time_in_absolute_time_units()); // Not sure why this is always 0, may be because it's on a simulated device.
 }
 
+// This test is disabled until the VM driver runtime is updated to include the new function (added in NI-SCOPE 2025 Q4).
```

Comment:
Consider using `GTEST_SKIP() << "Reasoning here.";` at the begging of the test instead of commenting it all out.

Reply:
That's interesting. I'll keep it in mind for the future.